### ISSUE TITLE
`[nav-item]` needs option for setting `role="presentation"` when wrapper `[nav]` is a `tablist`

### ISSUE THREAD

**Issue by cjg89** [UCF/Athena-Shortcodes-Plugin#87]
**Describe the bug**

In the following scenario:

```
[nav class="nav-tabs" tablist="true"]
[nav-item][nav-link id="tab-a" class="active" data_toggle="tab" href="#tab-pane-a"]Tab A[/nav-link][/nav-item]
[nav-item][nav-link id="tab-b" data_toggle="tab" href="#tab-pane-b"]Tab B[/nav-link][/nav-item]
[nav-item][nav-link id="tab-c" data_toggle="tab" href="#tab-pane-c"]Tab C[/nav-link][/nav-item]
[/nav]
```

The generated markup for this nav gets flagged in accessibility scans because each `.nav-item` has `role="listitem"` (implicit for `<li>`), and the parent `<ul>`'s role has been overridden to `role="tablist"`.

**Steps to Reproduce**

See above.

**Expected Behavior**

The inner `.nav-link`s should be detected as the inner `tab` elements for the nav, and the inbetween `.nav-item`s' roles should not interfere.

**Possible Fix**

Bootstrap addresses this in newer versions by applying `role="presentation"` to each `.nav-item`.  We may want to add some attribute to `[nav-item]` that sets `role="presentation"` on the generated `<li>` for these scenarios, or potentially adjust `[nav]`'s callback to detect and tweak inner `.nav-item`s when `tablist="true"`.

**Your environment:**

n/a

**Additional context**

n/a

**Comment by cjg89** [comment]
This was fixed a while ago: https://github.com/UCF/Athena-Shortcodes-Plugin/pull/83 I probably inadvertently opened this after seeing issues flagged in Siteimprove from markup added to pages with straight HTML (not shortcodes).  Closing.
